### ISSUE TITLE
Add to share sheet on Windows

### ISSUE THREAD

**Issue by Ahmed-bahaa77** [localsend/localsend#271]
### Is your feature request related to a problem? Please describe.

No, this feature exists in some platforms ex: Android. But we could extend that feature to windows for example.
There's native share sheet in windows 8/10/11
![A9AoRGY8wM](https://user-images.githubusercontent.com/79056295/224492909-bf7e8bdf-6f56-4306-ac18-5327612e625b.gif)

### Describe the solution you'd like

We can implement this feature using share_plus package or native solution.

**Comment by Tienisto** [comment]
The [share_plus](https://pub.dev/packages/share_plus) package is for sharing data **from** this app. But you want to receive share intents **to** the app.

LocalSend is currently using [share_handler](https://pub.dev/packages/share_handler) because it up-to-date.

**Comment by ShlomoCode** [comment]
Duplicate of https://github.com/localsend/localsend/issues/115

**Comment by ShlomoCode** [comment]
You are right... the other problem is only for the context menu, i.e. the explorer

**Comment by PandaFiredoge** [comment]
It doesn't seem to work?

**Comment by assiless** [comment]
I think they haven't released it yet `feat: support windows share sheet`, the release date is `Jul 9`, but latest release date is `Feb 20`

**Comment by PandaFiredoge** [comment]
> 我认为他们尚未发布 `feat: support windows share sheet` ，发布日期是 `Jul 9` ，但最晚发布日期是 `Feb 20`

https://localsend.ob-buff.dev/
